src/dbnode/storage/profiler/types.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2021 Uber Technologies, Inc.
```

Comment:
`src/dbnode/storage` doesn't seem like the best place for profiler code. Maybe move files under `src/dbnode/storage/profiler/` to `src/x/instrument`?